contracts/integration-tests/package.json

```diff
@@ -0,0 +1,99 @@
+{
+    "name": "@0x/contracts-integration-tests",
```

Comment:
Can we change the name of this package to `@0x/contracts-integrations`? I envision this being the package that houses custom bridge contracts, exchange wrappers, and extensions for external contracts.

Reply:
I actually thought about naming it this. I really like this!